Api exchanges

maurperez opened the pull request.

Commit:
Base de la pagina web creada

Aun faltan varios detalles a pulir

Commit:
Finalizacion de la pagina

Agregue validacion de inputs y manejo de errores de las promesas

Commit:
Utilizacion de jquery

Modificamos en codigo de js para lograr que la pagina funcione utilizando jquery

Commit:
Agrega etiqueta requerida por bootstrap

Commit:
Actualizando estilo de la pagina

Exchanges metidos dentro de una carta con una mejor lectura.
Pequeños cambios en el funcionamiento

asus753 commented:
![exchanges](https://user-images.githubusercontent.com/56839325/74119314-aa758a80-4b9d-11ea-8782-94926dc2bc32.png)

Commit:
Instalacion ESlint

Commit:
Creacion de pequeño test para cypress

Esto me va a permitir corroborar, con mucha mas rapidez, el correcto funcionamiento del codigo a la hora de dividirlo en modulos

fsodano commented on clase-12/tarea/tarea-exchanges/src/index.js:
```diff
@@ -0,0 +1,102 @@
+/// <reference types="jquery"/>
```
nice, te funciona? pensé que era jQuery

maurperez replied:
Sisi, asi me funciono perfecto

fsodano commented on clase-12/tarea/tarea-exchanges/src/index.js:
```diff
@@ -0,0 +1,102 @@
+    method: "GET",
+    url: "https://api.exchangeratesapi.io/latest",
+    success: respuesta => {
+        añadirBases(respuesta)
```
ojota con las ñ en el código, intentá evitarlas!

Commit:
Division del codigo en modulos

Se dividio el codigo en modulos y se hizo pequeños cambios en cuanto a funcionalidad e interfaz
